### PR TITLE
Update to ROM 2.0

### PR DESCRIPTION
**Pull request by hmadison**
* Make sure we include the gem.
* Update the way we set up and test rom for integration testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,15 +4,17 @@ env:
   - CODECLIMATE_REPO_TOKEN=ef69eb6a9e19987527e5010a02bbf71cd4ddc8a5137ba308f299817f56d06e7f
 script: "bundle exec rake spec"
 rvm:
-  - 2.0
```
Comment (beauby) [inline .travis.yml]: I know support has ended for 2.0 but why not put it in the `allow_failure` section as it is still quite fresh?

Comment (hmadison) [inline .travis.yml]: Won't even get past installing dependencies.

**Comment by solnic** [comment]
Merged. Thank you. Would you like to push a new release too?

**Comment by hmadison** [comment]
@solnic I don't have push rights on rubygems for that.

**Pull request merged**